src/desktop/apps/artwork/components/commercial/templates/price.jade

```diff
@@ -17,4 +17,14 @@ if artwork.sale_message || isPermanentCollection
 
     if showPriceLabel
       .artwork-commercial__shipping-info
-        | Shipping, tax, and service quoted by seller
+        - var noShippingInfo = !artwork.domesticShipping && !artwork.internationalShipping
+        - var complimentaryShipping = artwork.domesticShipping == '$0' && artwork.internationalShipping == '$0'
+        - var domesticShippingOnly = artwork.domesticShipping && artwork.domesticShipping != '$0' && !artwork.internationalShipping
+        if noShippingInfo
+          | Shipping, tax, and service quoted by seller
+        else if complimentaryShipping
+          | Free shipping worldwide
+        else if domesticShippingOnly
```

Comment:
I don't think we are reflecting in comps the situation that only domestic shipping is complimentary. I guess one could enter it by either leaving domestic shipping blank or 0 in the form and entering value for international shipping. Not clear if this case will be supported in CMS.

Reply:
Shouldn't all of this just live in Metaphysics? This seems like UI logic that will likely be shared among clients, MP should probably be in charge of returning 'Free shipping worldwide', 'Shipping, tax, and service quoted by seller', etc.

What do you think?

Reply:
My thinking was that we might want to have different strings there depending on the client. Or we want to introduce translations, etc.. Putting that much strings specific to force seemed a bit excessive in MP. Happy to update if you think having a full string for one `shippingString` is better.

Reply:
I think all those options could (and probably should) live in MP. That way they can easily be shared, we can add params if needed for different cases, they can be well encapsulated and tested, etc. The less 'logic' in Force + Reaction, and the more pure rendering of JSON, the better I think.

Reply:
Agreed @mzikherman. Once this refactor gets in https://github.com/artsy/metaphysics/pull/1207 - I'll update the code here.